articles/variables-in-js/index.md

```diff
@@ -0,0 +1,129 @@
+# Understanding Variables In Javascript
```

Comment:
Remove this heading. You should not use `#` and `##` for headings in the article.
Only `###` and up must be used.